<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsEditDialog</class>
 <widget class="QDialog" name="TagsEditDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit File Tags</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLineEdit" name="newTagLineEdit">
           <property name="maxLength">
            <number>50</number>
           </property>
           <property name="placeholderText">
            <string>New Tag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addTagButton">
           <property name="toolTip">
            <string>Add Tag</string>
           </property>
           <property name="text">
            <string notr="true">+</string>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="objectTagsTreeView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addTagsButton">
         <property name="toolTip">
          <string>Set Tag</string>
         </property>
         <property name="text">
          <string notr="true">&lt;&lt;</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeTagsButton">
         <property name="toolTip">
          <string>Unset Tag</string>
         </property>
         <property name="text">
          <string notr="true">&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLineEdit" name="tagsFilterLineEdit">
           <property name="maxLength">
            <number>50</number>
           </property>
           <property name="placeholderText">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearFilterButton">
           <property name="toolTip">
            <string>Clear Filter</string>
           </property>
           <property name="text">
            <string notr="true">X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="allTagsTreeView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TagsEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TagsEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
